content/plugins/dll-plugin.md

```diff
@@ -0,0 +1,145 @@
+* `scope` (optional): prefix which is used for accessing the content of the dll
+* `sourceType` (optional): how the dll is exposed ([libraryTarget][docs-libraryTarget])
+
+```javascript
+new webpack.DllReferencePlugin(options)
+```
+
+References a dll manifest file to map dependency names to module ids, then requires them as needed using the internal `__webpack_require__` function.
+
+W> Keep the `name` consistent with `output.library`.
+
+
+### Modes
+
+This plugin can be used in two different modes, _scoped_ and _mapped_.
+
+#### Scoped Mode
+
+The content of the dll is accessible under a module prefix. i.e. with `scope = "xyz"` a file `abc` in the dll can be access via `require("xyz/abc")`.
+
+T> [See an example use of scope][examples-dll-user]
+
+#### Mapped Mode
+
+The content of the dll is mapped to the current directory. If a required file matches a file in the dll (after resolving), then the file from the dll is used instead.
+
+Because this happens after resolving every file in the dll bundle, the same paths must be available for the consumer of the dll bundle. i.e. if the dll contains `lodash` and the file `abc`, `require("lodash")` and `require("./abc")` will be used from the dll, rather than building them into the main bundle.
+
+
+## Usage
+
+W> `DllReferencePlugin` and `DllPlugin` are used in _separate_ webpack configs.
+
+**webpack.vendor.config.js**
+
+```javascript
+new webpack.DllPlugin({
+  context: __dirname,
+  name: "[name]_[hash]"
+  path: path.join(__dirname, "manifest.json"),
+})
+```
+
+**webpack.app.config.js**
+
+```javascript
+new webpack.DllReferencePlugin({
+  context: __dirname,
+  manifest: require("./manifest.json"),
+  name: "./my-dll.js",
+  scope: "xyz",
+  sourceType: "commonsjs2"
+})
+```
+
+
+## Examples
+
+### [vendor][examples-dll-vendor] and [user][examples-dll-user]
+
+_Two separate example folders. Demonstrates scope and context._
+
+T> Multiple `DllPlugins` and multiple `DllReferencePlugins`.
+
+
+### Related
+
+* [code splitting example][examples-explicit-vendor-chunk]
+
+
+## References
+
+### Source
+
+* [DllPlugin source][src-DllPlugin]
+* [DllReferencePlugin source][src-DllReferencePlugin]
+* [DllModuleFactory source][src-DllModuleFactory]
+* [ManifestPlugin source][src-ManifestPlugin]
+
+### Tests
+
+* [DllPlugin creation test][tests-DllPlugin-0]
+* [DllPlugin without scope test][tests-DllPlugin-2]
+* [DllReferencePlugin use Dll test][tests-DllPlugin-1]
+
+
+[examples-examples]: https://github.com/webpack/webpack/tree/master/examples
+[examples-dll-vendor]: https://github.com/webpack/webpack/tree/master/examples/dll
+[examples-dll-user]: https://github.com/webpack/webpack/tree/master/examples/dll-user
+[examples-explicit-vendor-chunk]: https://github.com/webpack/webpack/tree/master/examples/explicit-vendor-chunk/README.md
+
+[src-DllReferencePlugin]: https://github.com/webpack/tree/master/lib/DllReferencePlugin.js
```

Comment:
@aretecode unfortunately our current setup doesn't support reference links:

![image](https://cloud.githubusercontent.com/assets/8988697/25732581/3b75c596-311f-11e7-814b-cf0a3e31ff07.png)

This is something I hope to resolve at some point #178, but for now can you inline these?